pgjdbc/src/main/java/org/postgresql/jdbc/PgResultSet.java

```diff
@@ -137,8 +137,8 @@ public ResultSetMetaData getMetaData() throws SQLException {
   }
 
   PgResultSet(Query originalQuery, BaseStatement statement, Field[] fields, List<Tuple> tuples,
-      ResultCursor cursor, int maxRows, int maxFieldSize, int rsType, int rsConcurrency,
-      int rsHoldability) throws SQLException {
+              ResultCursor cursor, int maxRows, int maxFieldSize, int rsType, int rsConcurrency,
```

Comment:
This appears to be just formatting, can you remove this change and the one below it ?

Reply:
Sure, I just push a change for this